AlexBeerShop_MVVM/Views/Controllers/SearchVC.swift

```diff
@@ -7,10 +7,90 @@
 
 
 import UIKit
+import SnapKit
+import Kingfisher
 
 
 final class SearchVC: UIViewController {
+    //MARK: - Elements
+    private let searchController = UISearchController(searchResultsController: nil)
+    private let beerViewTamplate = DetailBeerView()
+    private let beerViewModel = BeerViewModel()
```

Comment:
Single responsibility principle. Need to separate logic to different ViewModels. Don't use one VM for a few Views